### ISSUE TITLE
Enabling Caps Lock disables the Ctrl + Shift + N keyboard shortcut

### ISSUE THREAD

**Issue by KomalleanaByzantis** [elementary/files#1204]
# Prerequisites
- [x] I have searched open and closed issues for duplicates.

# Describe the error

Enabling Caps Lock disables the Ctrl + Shift + N keyboard shortcut which creates a new folder.

This seems related to issue #260, albeit with a different shortcut this time. Who knows if other shortcuts are affected? Nevertheless this is the only one which I have so far found to be disabled.

## To reproduce

1. Open Files.
2. Enable Caps Lock.
3. Press Ctrl + Shift + N.
4. A new folder is not created.

## Expected behaviour

A new folder is created when using the keyboard shortcut.

## Screenshots or screen recordings

[Screen recording](https://github.com/elementary/files/files/4140788/Enabling.Caps.Lock.disables.the.Ctrl.%2B.Shift.%2B.N.keyboard.shortcut.zip)

# Platform information

![System information](https://user-images.githubusercontent.com/51057001/73564096-2d406c00-445f-11ea-83e1-a7e47da88737.png)

- [ ] I'm using the latest version from git that I've manually compiled
- [x] I'm using the latest released stable version

Language: French
Keyboard: English (US) - English (Dvorak intl., with dead keys)
Location: Paris

The testing system is installed on a small partition on a ThinkPad T470 laptop on which UEFI Secure Boot is disabled. Aside from specifying the language, keyboard, location, initial user, and the specific partition in which to install elementary OS, all of the default settings were used during the installation. No Internet connection was used during the installation. The only system-wide changes which have been made after the installation are that the system connects to the Internet by wi-fi via a Linksys WRT54GL Wireless Router and that [Screen Recorder](https://appcenter.elementary.io/com.github.mohelm97.screenrecorder/) has been installed via AppCentre.

# Additional context

Before this test was performed, the following steps were followed:

1. All system-wide changes from previous tests were reverted.
2. All system components were updated via the blue “Update All” button in AppCentre.
3. A new administrator user named “Essai” or “Essai 2” was created with the password “komalleana”.
4. The current user was logged out and the new user was logged in.
5. The old user was deleted, the screen resolution was set to 800x600 at 60 Hz, tap clicking was disabled, and the touchpad pointer speed was set to the maximal value.
6. The computer was rebooted.
6. The test user was logged in.

**Comment by jeremypw** [comment]
Yes, need to review key press handling in general for other possible effects of CapsLock.

**Comment by jeremypw** [comment]
Looks like this was fixed for Copy/Paste but never got fixed for New Folder. Increasing priority.
